packages/block-editor/src/hooks/index.js

```diff
@@ -28,7 +28,8 @@ import contentLockUI from './content-lock-ui';
 import './metadata';
 import blockHooks from './block-hooks';
 import blockRenaming from './block-renaming';
-import './use-bindings-attributes';
+import './block-binding-support';
+// import './use-bindings-attributes';
```

Comment:
Instead of commenting it out, I think that we should just delete the current contents of `./use-bindings-attributes` and put the contents of `block-binding-support/with-block-binding-support` and `block-binding-support/index.js` there. 2 reasons:

1. The block bindings are not really special, they don't need to be in a separate folder.
2. The `with-block-binding-support` is not quite in line with the naming in the `/hooks` folder.

Reply:
I agree with this reasoning 🙂

Reply:
Definitely agree. We don't want lingering code that is unused.